tempesta_fw/http_parser.c

```diff
@@ -4180,6 +4185,13 @@ tfw_http_parse_resp(void *resp_data, unsigned char *data, size_t len)
 			/* Status code is fully parsed, move forward. */
 			resp->status = parser->_acc;
 			parser->_acc = 0;
+			/* Some responses are not allowed to have a body. */
+			/* TODO: Add (req == CONNECT && resp == 2xx) */
+			if (resp->status - 100U < 100U || resp->status == 204
```

Comment:
I'd say it's unclear now what's the reason for these checks. I would add an explicit reference to 7230 section 3.3.3 in the comment.